app/code/Magento/CmsGraphQl/Model/Resolver/DataProvider/Page.php

```diff
@@ -18,24 +20,40 @@
 class Page
 {
     /**
-     * @var FilterEmulate
+     * @var GetPageByIdentifierInterface
      */
-    private $widgetFilter;
+    private $pageByIdentifier;
 
     /**
      * @var PageRepositoryInterface
      */
     private $pageRepository;
 
     /**
-     * @param PageRepositoryInterface $pageRepository
+     * @var StoreManagerInterface
+     */
+    private $storeManager;
+
+    /**
+     * @var FilterEmulate
+     */
+    private $widgetFilter;
+
+    /**
+     * @param GetPageByIdentifierInterface $getPageByIdentifier
      * @param FilterEmulate $widgetFilter
+     * @param PageRepositoryInterface $pageRepository
+     * @param StoreManagerInterface $storeManager
      */
     public function __construct(
```

Comment:
You have changed `\Magento\CmsGraphQl\Model\Resolver\DataProvider\Page::__construct` params not according to `backward-compatible-development` guide.
Please check the correct way in this article: https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/
**section** "Adding a constructor parameter"

Reply:
**fixed**

Thank you!